### ISSUE TITLE
It freezes halfway through the dump, has no response, and cannot dump the partition.

### ISSUE THREAD

**Issue by xiaojie0815** [bkerler/edl#493]
![image](https://github.com/bkerler/edl/assets/46600752/28895c2c-5fd1-485d-ae39-7b2b1b0e9ca9)
It freezes halfway through the dump, has no response, and cannot dump the partition.

**Comment by xiaojie0815** [comment]
At this point, the progress bar stops moving and stays stuck here, unresponsive.

**Comment by RenateUSB** [comment]
It's probably a USB error that is not being detected and reported.
Have you tried this again? Does it stop at the exact same place?

**Comment by xiaojie0815** [comment]
> It's probably a USB error that is not being detected and reported. Have you tried this again? Does it stop at the exact same place?

Yes, copying each partition will get stuck. I am using a vm virtual machine.

**Comment by RenateUSB** [comment]
> I am using a vm virtual machine.

Well don't. VMs don't have a good reputation of working with USB.
Besides this is Python. Just run it on your naked machine.
